packages/bun-polyfills/src/global/index.ts

```diff
@@ -12,13 +20,43 @@ Blob.prototype.json = async function json<T>(this: Blob): Promise<T> {
         throw err;
     }
 };
+Blob.prototype.formData = async function formData(this: Blob): Promise<FormData> {
+    if (this.type.startsWith('multipart/form-data;')) {
+        return new Response(this.stream(), {
+            headers:
+                //? Good one Node: https://github.com/nodejs/node/issues/42266
+                { 'Content-Type': this.type.replace('webkitformboundary', 'WebkitFormBoundary') }
+        }).formData() as Promise<FormData>;
+    } else if (this.type === 'application/x-www-form-urlencoded') {
+        return readableStreamToFormData(this.stream());
+    } else {
+        throw new TypeError('Blob type is not well-formed multipart/form-data or application/x-www-form-urlencoded');
+    }
+};
+Reflect.set(Blob.prototype, 'readable', undefined /*satisfies BunFile['readable']*/);
+Reflect.set(Blob.prototype, 'lastModified', -1 satisfies BunFile['lastModified']);
+Reflect.set(Blob.prototype, 'exists', (async function exists() {
```

Comment:
Reflect.set is just the verbose way of a direct property assignment here.

Reply:
Reflect.set avoids triggering typescript about non-existent or type-mismatched fields, and also gracefully fails silent instead of throwing if the properties have been previously tampered with somehow (e.g. an earlier loaded package) in a way that causes the assignment to fail like making it non-writable.